packages/remix-server-runtime/routeModules.ts

```diff
@@ -59,8 +40,6 @@ export type ActionFunction = (
 export type ActionFunctionArgs = RRActionFunctionArgs<AppLoadContext> & {
   // Context is always provided in Remix, and typed for module augmentation support.
   context: AppLoadContext;
-  // TODO: (v7) Make this non-optional once single-fetch is the default
-  response?: ResponseStub;
```

Comment:
Remove the previous ambiguity here.  Now users will get no `response` stub when using `LoaderFunctionArgs` and a non-optional `response` stub when using `defineLoader`